### PR TITLE
Epmlstrcmw 28 float to big decimal

### PR DESCRIPTION
**Pull request by SergeySdv**
Add Changed type Double to BigDecimal

### CODE REVIEW

**Review by svitkovsergey — APPROVED** [review]
Nicely done! 👍 
Please squash commits before pushing it to EPAM's repository branch.

File: wom/src/main/scala/wom/types/WomFloatType.scala [hunk 1]
```diff
@@ -9,6 +9,7 @@ case object WomFloatType extends WomPrimitiveType {
   val stableName: String = "Float"
 
   override protected def coercion = {
+    case b: BigDecimal => WomFloat(b)
```
Comment (svitkovsergey) [inline wom/src/main/scala/wom/types/WomFloatType.scala:12]: I think that `bd` would be more appropriate here.

**Review by iliaonishchenko — APPROVED** [review]
Nice

**Comment by svitkovsergey** [comment]
Hi @SergeySdv @ilyaonishenko !
I think that since PR to broadinstitute was already opened, this one could be closed.

**Pull request closed without merging**